modules/ols-activating-token-quota-limits.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ols-activating-token-quota-limits_{context}"]
+= Activating token quota limits
+
+Activate token quota limits for the {ols-long} Service by defining key-value pairs in the `ConfigMap` resource. The {ols-long} pod mounts the `ConfigMap` resource as a volume, enabling access to the file stored within it. The `OLSConfig` Custom Resource (CR) references the `ConfigMap` resource to obtain the quota limit information.
+
+.Prerequisites
+
+* You have installed the the {ols-long} Operator.
+
+* You have configured a large language model provider (LLM).
+
+* A PostgreSQL database is configured and the {ols-long} Service can access the database.
+
+.Procedure
+
+. Open the {ols-long} `OLSconfig` CR file by running the following command:
++
+[source,terminal]
+----
+$ oc edit olsconfig cluster
+----
+
+. Modify the `data` property of the `spec.ols.quotaHandlersConfig` specification to include token quota limit information.
++
+.Example {ols-long} `OLSconfig` CR
+[source,yaml]
+----
+apiVersion: ols.openshift.io/v1alpha1
+kind: OLSConfig
+metadata:
+  name: cluster
+spec:
+  ols:
+    quotaHandlersConfig:
+      limitersConfig:
+      - name: user_limits 
+        type: user_limiter
+        initialQuota: 100000 <1>
+        quotaIncrease: 10000 <2>
+        period: 30 days
+      - name: cluster_limits <3>
+        type: cluster_limiter <4>
+        initialQuota: 1000000
+        quotaIncrease: 100000
+        period: 30 days <5>
+----
+<1> Specifies a token quota limit of 100,000 for each user over a period of 30 days.
+<2> Increases the token quota limit for the user by 1,000 over a period of 30 days.
+<3> Specifies a token quota limit of 1,000,000 for each cluster over a period of 30 days.
+<4> Increases the token quota limit for the cluster by 100,000 over a period of 30 days.
+<5> Defines the amount of time that the scheduler waits and then checks if the period interval is over. When the period interval is over, the scheduler stores the timestamp and resets or increases the quota limit.
+
```

Comment:
@JoaoFula Part of GA test , could you please confirm this 5 configs are working as documented here ?

Reply:
I can confirm that things are written in this way and that we only accept the correct type/ strings/ no negative numbers, etc. And make sure that after setting the quota to 10000, that the available quota is indeed 10000. However, I can't actually verify the quota increase or quota reset after the period because the minimum period is 1 day and I can only have clusters for a max of 12 hours.
But I'm 99% sure we have unit tests for that part and the period is configurable for a shorter time on the service side of things so this is definitely tested.